excludeFindBugsFilter.xml

```diff
@@ -7,8 +7,5 @@
     <!-- this problem is to be addressed in ZOOKEEPER-3227 -->
     <Bug pattern="DM_DEFAULT_ENCODING"/>
 
-    <!-- not really a problem -->
-    <Bug pattern="DM_EXIT"/>
```

Comment:
know we have only one place in code allowed to call directly System.exit, so it is better to enable the rule and suppress it locally